amp-update-cache/public/index.html

```diff
@@ -150,7 +150,7 @@
         <li>Generate RSA Keypairs, and make the public key available, as explained in the <a href="https://developers.google.com/amp/cache/update-ping#rsa-keys">Generate RSA keys</a> section of the documentation</li>
         <li>Paste the <em>AMP Cache Url</em> that is desired to be flushed.</li>
         <li>Paste the <em>private key</em> that matches the published public key on the respective field.</li>
-        <li>Click <em>Generate Url</em>. A link will be generated below the form</li>
+        <li>Click <em><a href="https://github.com/ampproject/amp-publisher-sample/amp-update-cache/cache-refresh.js">Generate Url</a></em>. A link will be generated below the form</li>
```

Comment:
Removed the link for now. Will add a link to the project on Github in the header or in the footer later.